config/rbac/role.yaml

```diff
@@ -11,8 +11,12 @@ rules:
   resources:
   - secrets
   verbs:
+  - create
+  - delete
   - get
   - list
+  - patch
```

Comment:
Is Delete and Patch required for secrets ?

Reply:
good catch! delete is required, patch can be removed. They are needed in order to allow the ClusterEnvironment controller to grant this permission to service agent's Service Account. However, we are not using Patch, so will remove it both from here and from Service Agent's requirements (i.e., `config/rbac/reporter_role.yaml`).

```
2023-04-26T12:30:00Z    ERROR   error binding namespace {"controller": "clusterenvironment", "controllerGroup": "primaza.io", "controllerKind": "ClusterEnvironment", "ClusterEnvironment": {"name":"worker","namespace":"primaza-system"}, "namespace": "primaza-system", "name": "worker", "reconcileID": "047815dc-0b32-4233-855b-c19629a839b0", "cluster-environment": "worker", "namespace": "services", "error": "rolebindings.rbac.authorization.k8s.io \"primaza-reporter-worker-services\" is forbidden: user \"system:serviceaccount:primaza-system:primaza-controller-manager\" (groups=[\"system:serviceaccounts\" \"system:serviceaccounts:primaza-system\" \"system:authenticated\"]) is attempting to grant RBAC permissions not currently held:\n{APIGroups:[\"\"], Resources:[\"secrets\"], Verbs:[\"create\" \"delete\" \"patch\" \"update\"]}"}
```